app/views/partials/trials-list.html

```diff
@@ -158,11 +158,11 @@ <h4 title="{{ trial.title }}">{{ trial.title }}</h4>
               </dl>
             </li>
             {% endif %}
-            {% if trial.sponsors.length > 0 %}
+            {% if trial.fundedBy.length > 0 %}
             <li>
               <dl class="sponsors">
                 <dt>Sponsored by</dt>
-                <dd>{{ joinListOfNames(trial.sponsors) | safe }}</dd>
+                <dd>{{ joinListOfNames(trial.fundedBy) | safe }}</dd>
```

Comment:
I renamed all the properties that weren't a transcript of fields from the original data source (Google Docs). So everything is now under the original name, converted from snake_case to camelCase. Easier to follow and cross check with the original.